extract-wos-article.py

```diff
@@ -23,7 +30,54 @@
     "Current Publisher": "publisher",
 }
 
-YEAR_REGEX = re.compile(".*([0-9]{4})")
+REGEX = {
+    "year": re.compile(".*([0-9]{4})"),
+    "wos_id_number": re.compile("([0-9]{15})"),
+}
+CITING_SUMMARY_URL_PREFIX = "https://www.webofscience.com/wos/woscc/citing-summary/"
```

Comment:
I guess this will be a bit longer than 80 characters of PEP8's default -- I'd be happy to go for something like a 100 though :slightly_smiling_face:

Reply:
@mrshu just to make it clear for me, you'd happy to set 100 over an entire project?

Reply:
@JMatej Right, that's what I meant -- sorry for not being clear about it